Document/0x06g-Testing-Network-Communication.md

```diff
@@ -212,7 +223,7 @@ In that case, the certificate associated or 'pinned' to the host at when it seen
 
 #### When the Pin Fails
 
-Pinning failures can occur for various reasons: either the app is expecting another key or certificate than offered by the server and/or load balancer, or there might be a man-in-the-middle attack going on. In both cases and similar as with Android, there are various ways to respond to such a situation. Please see the section "[When the Pin Fails](0x05g-Testing-Network-Communication.md#when-the-pin-fails)" in the chapter "Android Network APIs".
+Pinning failures can occur for various reasons: either the app is expecting another key or certificate than offered by the server and/or load balancer, or there might be a man-in-the-middle attack going on. In both cases and similar as with Android, there are various ways to respond to such a situation. Please see the section "[When the Pin Fails](0x05g-Testing-Network-Communication.md#when-the-pin-fails)" in the chapter "Android Network Communication".
```

Comment:
```suggestion
Pinning failures can occur for various reasons: either the app is expecting another key or certificate than offered by the server, or there might be a man-in-the-middle attack occuring. In both cases and similar as with Android, there are various ways to respond to such a situation. Please see the section "[When the Pin Fails](0x05g-Testing-Network-Communication.md#when-the-pin-fails)" in the chapter "Android Network Communication".
```
I think the mentioning of a loadbalancer is unnecessary here, and it would require us to mention it in many other places as well. I think it makes sense to view the external targeted system as the 'server' as a generic name.

Reply:
I totally agree!